docs/user-guide/personalidentifiableinformationidentificationandremoval.rst

```diff
@@ -241,7 +268,147 @@ For example:
+            }
+        }
+    }
+
+For reference, the default system prompt is:
+
+.. code-block:: text
+
+    "You are an expert redactor. The user is going to provide you with some text. 
+    Please find all personally identifying information from this text. 
+    Return results according to this JSON schema: {JSON_SCHEMA}
+    Only return results for entities which actually appear in the text. 
+    It is very important that you return the entity_text by copying it exactly from the input. 
+    Do not perform any modification or normalization of the text. 
+    The entity_type should be one of these: {PII_LABELS}"
+
+``{PII_LABELS}`` represents a comma-separated list of strings corresponding to the PII entity types you want to identify (e.g., "name", "email", "ip_address", etc.).
+
+When using a custom system prompt with non-English text, make sure to adapt the instructions while maintaining the exact JSON schema requirement. The LLM models will use this system prompt to guide their identification of PII entities.
+
+############################
+Multilingual PII Redaction
+############################
+
+By default, NeMo Curator's PII redaction tools are configured to work with English text. When processing multilingual datasets, you'll need to explicitly configure the PII tools to support additional languages.
+
+Method
+~~~~~~~~~~~~~~~~~~~~~~
+
+The most common way in which PII redaction is used in NeMo Curator is either directly using ``PiiDeidentifier`` or with modifiers:
+
+.. code-block:: python
+
+    deidentifier = PiiDeidentifier()
+    deidentifier.deidentify_text("Hello, my name is John Smith. Email: john.smith@example.com")
+
+.. code-block:: python
+
+    modifier = PiiModifier(
+        supported_entities=["PERSON", "EMAIL_ADDRESS", "PHONE_NUMBER"],
+        anonymize_action="replace",
+        new_value="[REDACTED]", 
+        batch_size=1000,
+        device="cpu"
+    )
+
+    input_file = "./data/sample.jsonl"
+    source_data = read_data([input_file], file_type="jsonl", backend="pandas", 
+    add_filename=True)
+    dataset = DocumentDataset(source_data)
+    modify = Modify(modifier)
+    modified_dataset = modify(dataset)
+
+The previous code makes the implicit assumption that the language of the text documents is English. If a piece of text is in another language, the current implementation will fail:
+
+.. code-block:: python
+
+    deidentifier = PiiDeidentifier(language="ja")
+    deidentifier.deidentify_text("私の名前は田中です。メールは tanaka@example.jp です。")
+
+This will raise the following error:
+
+.. code-block:: text
+
+    ValueError: No matching recognizers were found to serve the request.
+
+Solution
+~~~~~~~~~~~~~~~~~~~~~~
+
+To enable multilingual PII redaction, modify lines ``99-102`` in ``algorithm.py`` by passing an additional argument ``models`` to ``CustomNlpEngine``. This argument must be a list of dictionaries, where each dictionary contains 2 key-value pairs: ``lang_code`` and ``model_name``.
```

Comment:
I didn't realize this solution involves editing source code. I don't think we should include this, especially since line numbers can change with commits. A better approach would be to hold off on this section for now until we can edit the source code to implicitly handle multilingual PII. What do you think?

Reply:
sure, that sounds good to me

Reply:
Opened https://github.com/NVIDIA/NeMo-Curator/issues/661 for now. Thanks!